Library/Homebrew/vendor/gems/mechanize-2.9.1/mechanize-2.9.1

```diff
@@ -0,0 +1 @@
+mechanize-2.9.1/
```

Comment:
@dduugg, this symlink was removed in https://github.com/Homebrew/brew/pull/15468, but `brew vendor-gems` recreated it. Didn't we remove `mechanize` from being vendored automatically?

Reply:
AFK traveling for the next couple weeks, but the symlink approach was authored by @MikeMcQuaid IIRC

Reply:
@reitermarkus https://github.com/Homebrew/brew/blob/4446fe603cc19316036fd07e5df1124c4d15ead2/Library/Homebrew/dev-cmd/vendor-gems.rb#L57-L70 is the relevant code that would need changed/fixed here. The symlink approach is so we can maintain some idea of what version these files came from and use `gem` to install/update them.